📦 Missing dependency: qrcode.react

## Problem
`TicketCard` component imports `qrcode.react` which is not installed.

## Error
```
src/components/TicketCard.tsx(8,25): error TS2307: Cannot find module 'qrcode.react'
```

## Impact
- Severity: HIGH - P1
- Type-checking fails
- QR code functionality broken

## Fix
```bash
npm install --workspace=@base-ticketing/web qrcode.react
```

--- a/apps/web/package.json
+++ b/apps/web/package.json
@@ -21,6 +21,7 @@
     "lucide-react": "^0.400.0",
     "moment": "^2.30.1",
     "next": "^14.0.0",
+    "qrcode.react": "^4.2.0",
     "react": "^18.2.0",
     "react-big-calendar": "^1.19.4",
     "react-dom": "^18.2.0",

